### ISSUE TITLE
Sprint colouring doesn't update on re-order

### ISSUE THREAD

**Issue by MarkMaldaba** [bayoteers/AgileTools#25]
In the planning view, when you drag items from the backlog into the current sprint, the colouring which indicates capacity is automatically updated.  However, when you re-order items in the sprint itself, the colours don't update.

It would be good if these updated in real-time, without requiring a refresh.

**Comment by keto** [comment]
Yes, this should be quite simple fix. In scrum.js adding _onMove() handler which calls _calculateWork() in the SprintController should fix it.
